CMSgov/design-system
# [NO TICKET] Update Accordion vis-reg ref images

## Summary
No ticket for this work.

In removing all the util classes out of our components, I noticed our reference images were outdated for the Accordion component. And since the Accordion didn't use any util classes, I didn't have a story to bundle this ref image update work with, so here we are.

The outdated Accordion references looked like the font-weight had changed slightly over time. The Backstop report showed <2% difference between the reference and current component.

## Testing
To test... you could pull this branch down and run the following commands, then run a similar set of commands against  `master` to see what Backstop returns in each branch:

1. Pull down branch
2. `nvm use`
3. `yarn install && yarn start`
4. Wait 3.7 years while project builds
5. In another tab in your terminal, run `yarn backstop test`
6. Wait another 1.2 years for diff to generate

Here's a screenshot of what the before ref looked like:
![Screen Shot 2021-10-11 at 3 52 33 PM](https://user-images.githubusercontent.com/5159392/136847960-69d0d25a-5389-47c7-a430-31f390c0de15.png)
